### ISSUE TITLE
Add mlf-core to conda

### ISSUE THREAD

**Issue by Zethson** [mlf-core/mlf-core#242]
We are currently only distributing mlf-core to PyPI. I would be very happy if we could also make it available on Conda. This requires us to first bring system-intelligence to Conda which may even block us completely. My initial attempts were not that successful. 

Will see what happens though :)

**Comment by KevinMenden** [comment]
That would be great ... haven't had the best of experiences with conda so far :/ at least for bioconda I continue to run into issues with tensorflow.

But hey I'm sure it'll work better here 😁

**Comment by Zethson** [comment]
> That would be great ... haven't had the best of experiences with conda so far :/ at least for bioconda I continue to run into issues with tensorflow.
> 
> But hey I'm sure it'll work better here grin

Yeah my experiences were horrible a well haha. mlf-core has so many dependencies that I worry that we will have to add a couple of them ourselves first to Conda. And those packages themselves may require other Conda packages first :scream:

**Comment by KevinMenden** [comment]
Alright I will de-assign myself 🤣

**Comment by Zethson** [comment]
Actually it doesn't look that bad:

```
cookiecutter==1.7.2
click==7.1.2
ruamel.yaml==0.16.12
pygithub==1.54.1
GitPython==3.1.12
autopep8>=1.5.4
cffi==1.14.4
cryptography==3.3.1
requests>=2.24.0
rich==9.10.0
packaging==20.9
appdirs==1.4.4
questionary==1.9.0
pynacl==1.4.0
mlflow
```

Most of them I already know of that they are on Conda. The others we will have to look up. This would be the first step.

We can also get rid of autopep8 and just use black for the templates.

**Comment by KevinMenden** [comment]
Yes that looks indeed doable 👍 
Have only put stuff on bioconda so far - the process is similar for conda-forge I guess?
No idea about the anaconda channel
